[2025-02-23T14:22:23.349+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T14:22:23.380+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dubai_weather_pipeline.load_weather_data manual__2025-02-23T14:22:17.650371+00:00 [queued]>
[2025-02-23T14:22:23.393+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dubai_weather_pipeline.load_weather_data manual__2025-02-23T14:22:17.650371+00:00 [queued]>
[2025-02-23T14:22:23.393+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T14:22:23.424+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): load_weather_data> on 2025-02-23 14:22:17.650371+00:00
[2025-02-23T14:22:23.430+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=19773) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T14:22:23.431+0000] {standard_task_runner.py:72} INFO - Started process 19774 to run task
[2025-02-23T14:22:23.432+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dubai_weather_pipeline', 'load_weather_data', 'manual__2025-02-23T14:22:17.650371+00:00', '--job-id', '131', '--raw', '--subdir', 'DAGS_FOLDER/dubai_weather_pipeline/dubai_weather_etl.py', '--cfg-path', '/tmp/tmp8p05hxih']
[2025-02-23T14:22:23.433+0000] {standard_task_runner.py:105} INFO - Job 131: Subtask load_weather_data
[2025-02-23T14:22:23.498+0000] {task_command.py:467} INFO - Running <TaskInstance: dubai_weather_pipeline.load_weather_data manual__2025-02-23T14:22:17.650371+00:00 [running]> on host ee2081ae3a04
[2025-02-23T14:22:23.635+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dubai_weather_pipeline' AIRFLOW_CTX_TASK_ID='load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T14:22:17.650371+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-23T14:22:17.650371+00:00'
[2025-02-23T14:22:23.636+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T14:22:23.654+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2025-02-23T14:22:23.668+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-23T14:22:23.686+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T14:22:23.686+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dubai_weather_pipeline, task_id=load_weather_data, run_id=manual__2025-02-23T14:22:17.650371+00:00, execution_date=20250223T142217, start_date=20250223T142223, end_date=20250223T142223
[2025-02-23T14:22:23.766+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T14:22:23.792+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-23T14:22:23.795+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
